### PR TITLE
some cleanup

### PR DESCRIPTION
**Pull request by maxbachmann**

### CODE REVIEW

**Comment by maxbachmann** [comment]
mainly did some small restructuring to reduce the if/else depth.

Fixed return value of onMessage, that was always true -> would accept any intent, since onMessage is called as fallback by every intent that is not ment for AliceCore

File: PublishedModules/ProjectAlice/AliceCore/AliceCore.py [hunk 1]
```diff
@@ -473,15 +466,14 @@ def confirmModuleReboot(self, session: DialogSession, **_kwargs):
...
-		if commons.isYes(session):
-			value = 'greetAndRebootModules'
+		value = 'greetAndRebootModules' if commons.isYes(session) else 'greet'
 
 		self.ConfigManager.updateAliceConfiguration('onReboot', value)
 		self.endDialog(session.sessionId, self.randomTalk('confirmRebooting'))
 		self.ThreadManager.doLater(interval=5, func=subprocess.run, args=[['sudo', 'shutdown', '-r', 'now']])
 
 
+	#TODO return value should be a dict according to Module.py, so it appears something is wrong here
```
Comment (maxbachmann) [inline PublishedModules/ProjectAlice/AliceCore/AliceCore.py]: I did not change this, but I think it is not correct with the new dialog mapping anymore. I think it should return ```self.supportedIntents```

Comment (Psychokiller1888) [reply PublishedModules/ProjectAlice/AliceCore/AliceCore.py]: Looks wrong yes

**Comment by maxbachmann** [comment]
fixes a couple of places where endDialog is used multiple times

**Pull request merged**